package.json

```diff
@@ -163,6 +163,7 @@
     "qrcode.react": "0.8.0",
     "react": "16.6.3",
     "react-copy-to-clipboard": "5.0.1",
+    "react-custom-scrollbars": "^4.2.1",
```

Comment:
@yakovkaravelov please set a static version here by removing `^`...

Reply:
You also need to re-run the app after you change this so that the `yarn.lock` gets updated @yakovkaravelov

Reply:
@nikolaglumac Just updated version flag.